options/locale/locale_bg-BG.ini

```diff
@@ -365,7 +365,7 @@ pulls.new=Нова заявка за сливане
 pulls.filter_branch=Филтър по клон
 pulls.no_results=Няма резултати.
```

Comment:
Only en-US translation should be committed.

Reply:
Ah right, translations are handled via CrowdIn. In order to change the target branch name on the already loaded page I'd need to assign a ID to the translation string (pulls.title_desc) like this: `<code id="branch_target">%[3]s</code>`

Apart from the newly introduced string in the English (and German) translation file that's the only change to the translations I've made. I'll remove these changes later and resubmit them via CrowdIn.